src/sentry/static/sentry/app/components/alertMessage.jsx

```diff
@@ -1,9 +1,37 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import styled from 'react-emotion';
 
+import Alert from '../components/alert';
 import AlertActions from '../actions/alertActions';
+import InlineSvg from '../components/inlineSvg';
 import {t} from '../locale';
 
+const StyledAlert = styled(Alert)`
+  padding: ${p => p.theme.grid}px ${p => p.theme.grid * 2}px;
+  position: relative;
+  margin: 0;
+  padding-right: 30px;
+
+  .close {
```

Comment:
Ok here is my most annoying comment of the review: this probably shouldn't be scoped in the css since the element is inside the scope of this component. something like: 

```jsx
const StyledCloseButton = styled((...props) => (
  <button
     {...props}
     type="button"
     className="close"
     aria-label={t('Close')}
     onClick={this.closeAlert}
     ><InlineSvg aria-hidden="true" src="icon-circle-close" />
  </button>
))`
  //styles go here
`
```
Is (to the best of my experience) the right thing to do, even though it is a bit longer. If that is confusing slash weird lemme know and we can hash it out together.

Reply:
Agreed! Just thinking out loud here, I think in ideal world we probably want a CloseButton here (is "className" needed? for bootstrap?). CloseButton would have below styles except for positioning.


And `StyledAlert` would look something like this since positioning is unique to Alert component:

```
const StyledAlert = styled(Alert)`
  padding: ${p => p.theme.grid}px ${p => p.theme.grid * 2}px;
  position: relative;
  margin: 0;
  padding-right: 30px;

  ${CloseButton} {
    position: absolute;
    right: 8px;
    top: 7px;
  }
```

Reply:
Cool, will update to this. 

P.s. I don't think "className" is doing anything here at all and should be removed.